### DISCUSSION TITLE
How to redeploy cognito.yaml

### DISCUSSION THREAD

**Discussion by tuhin24** [aws-samples/aws-saas-factory-ref-solution-serverless-saas#71] [Q&A] [answered]
Hi,
I added another custom attribute in cognito.yaml. How do I redeploy it. The pipeline does not seem to deploy the changes automatically. Thanks

**Comment by ujwalbukka** [comment, marked as answer]
Hi @tuhin24 - cognito.yaml is part of the [bootstrap-template.yaml](https://github.com/aws-samples/aws-saas-factory-ref-solution-serverless-saas/blob/main/server/bootstrap-template.yaml), you just need to redeploy it. You can deploy it in below ways

1. You can run command at [line54](https://github.com/aws-samples/aws-saas-factory-ref-solution-serverless-saas/blob/main/deployment.sh#L54) and [line55](https://github.com/aws-samples/aws-saas-factory-ref-solution-serverless-saas/blob/main/deployment.sh#L55) from deployment.sh
2. You can also just re-run whole deployment.sh too.
